lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -81,12 +95,33 @@ def active_step_by_step
         @active_step_navs.first
       end
 
+      def secondary_step_by_steps
+        secondary_to_step_navs
+      end
+
+      def secondary_step_by_step?
+        secondary_step_by_steps.any?
+      end
+
+      def secondary_step_by_step
+        secondary_to_step_navs.first
+      end
+
+      def show_secondary_step_by_step?
+        step_navs_combined_list.count === 0 && secondary_step_by_step? && secondary_step_by_steps.count === 1
```

Comment:
I think for ease of reading, `step_navs_combined_list.count === 0` could be moved into the `secondary_step_by_step?` method, e.g:

`step_navs_combined_list.count === 0 && secondary_step_by_steps.any?`